src/main/java/org/jenkinsci/plugins/xunit/threshold/QuarantinedFilesFileFilter.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.xunit.threshold;
+
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.commons.io.filefilter.IOFileFilter;
+import java.io.File;
+import java.io.Serializable;
+
+
+// the filter class needs to be serializable to work for remote workspace searches
+public class QuarantinedFilesFileFilter extends DirectoryFileFilter implements Serializable {
+    public static final IOFileFilter DIRECTORY = new QuarantinedFilesFileFilter();
+    public static final IOFileFilter INSTANCE;
+    static final String QUARANTINED_TEST_FILE = "quarantined-tests.json";
```

Comment:
I think it'd be better to pass this filename in, especially since we have it here and in the other class or have it somewhere accessible and visible to both. It also makes refactoring easier if someone decides to change the file for any reason

Reply:
yep fixed. I wasn't sure if a private gets serialized but refactored - tested and it worked!